src/NHibernate.Test/NHSpecificTest/GH1228/Fixture.cs

```diff
@@ -7,126 +7,95 @@ public class Fixture : BugTestCase
 		[Test]
 		public void TestOk()
 		{
-			using (ISession s = OpenSession())
+			using var s = OpenSession();
 			{
-				using (ITransaction t = s.BeginTransaction())
-				{
-					try
-					{
-						{
-							var queryThatWorks = s.CreateQuery(@"
+				var queryThatWorks = s.CreateQuery(
+					@"
 							SELECT ROOT FROM NHibernate.Test.NHSpecificTest.GH1228.Sheet AS ROOT 
 							WHERE (EXISTS (FROM NHibernate.Test.NHSpecificTest.GH1228.Shelf AS inv  
 										   , ROOT.Folder AS ROOT_Folder 
 										   WHERE ROOT_Folder.Shelf = inv AND inv.Id = 1
 								  )	   ) 
 								  AND ROOT.Name = 'SomeName'");
-							queryThatWorks.List();
-						}
-						{
-							var queryThatWorks = s.CreateQuery(@"
+				queryThatWorks.List();
+			}
+			{
+				var queryThatWorks = s.CreateQuery(
+					@"
 							SELECT ROOT FROM NHibernate.Test.NHSpecificTest.GH1228.Shelf AS ROOT 
 							WHERE (EXISTS (FROM NHibernate.Test.NHSpecificTest.GH1228.Sheet AS sheet
 										   , ROOT.Folders AS ROOT_Folder 
 										   WHERE ROOT_Folder = sheet.Folder AND sheet.Name = 'SomeName'
 								  )	   ) 
 								  AND ROOT.Id = 1");
-							queryThatWorks.List();
-						}
-					}
-					finally
-					{
-						s.Delete("from Sheet");
-						s.Delete("from Folder");
-						s.Delete("from Shelf");
-						t.Commit();
-					}
-				}
+				queryThatWorks.List();
 			}
```

Comment:
End of resulting changes:

```suggestion
			queryThatWorks.List();
```